src/universal/modules/summary/containers/MeetingSummary/MeetingSummaryContainer.js

```diff
@@ -99,20 +100,56 @@ const mapStateToProps = (state, props) => {
   };
 };
 
-const MeetingSummaryContainer = (props) => {
-  return <MeetingSummary {...props} />;
-};
+@requireAuth
+@connect(mapStateToProps)
+export default class MeetingSummaryContainer extends Component {
+  static propTypes = {
+    actionCount: PropTypes.number,
+    agendaItemsCompleted: PropTypes.number,
+    dispatch: PropTypes.func,
+    meetingNumber: PropTypes.number,
+    projectCount: PropTypes.number,
+    teamId: PropTypes.string,
+    teamMembers: PropTypes.array,
+    teamName: PropTypes.string
+  };
+
+  componentWillMount() {
+    this.state = {segmentEventsSent: false};
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const {segmentEventsSent} = this.state;
+    const {dispatch, meetingNumber} = nextProps;
+    if (!meetingNumber || segmentEventsSent) return;
+    /*
+     * Events to track:
+     *
+     * (1) team has concluded their first meeting
+     * (2) team has concluded their second meeting
+     *
+     * N.B. it is ok if these are sent as dupes when viewed from meeting history
+     */
+    if (meetingNumber === 1) {
```

Comment:
shouldn't this be a side effect that comes from `endMeeting` on the server? Dispatching from the client will give us a bunch of false positives since folks could refresh the page or view the meeting from the meeting history